src/test/resources/xml/copyright_extraction/09222b40-dba1-11e5-9785-0016357f605f.xml

```diff
@@ -45,17 +45,17 @@
                             <mods:languageTerm authority="rfc4646" type="code">da</mods:languageTerm>
                         </mods:languageOfCataloging>
                     </mods:recordInfo>
-                    <mods:relatedItem type="otherFormat">
-                        <mods:identifier displayLabel="image" type="uri">http://www.kb.dk/imageServicecumulus-core-01:/Depot/DAM/Samlingsbilleder/Samlingsbilleder/samlingsbilleder/KOB/Elfelt_4/Formidlingsfiler/Elf_113136.jpg</mods:identifier>
-                    </mods:relatedItem>
+                    <mods:titleInfo>
```

Comment:
Shouldn't there be an image associated with this metadata record?

Reply:
Yes, ideally! However as I have just re-extracted the metadata record the field is still missing.